test/server/chat-plugins/hosts.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Tests for the Hosts chat plugin
```

Comment:
I wonder if any functionality that needs testing should just be moved into `ip-tools`...

Reply:
All of this is stuff that could plausibly go in `ip-tools`, to me.

Reply:
I think that the visualization function, at least, makes more sense in the chat plugin. Sorting and checking for conflicts can be moved if you want though.